frontend/templates/views/partials/analytics.j2

```diff
@@ -9,6 +9,20 @@
         }
       }
     }
+{% if '/courses/' in doc.pod_path and doc.title == 'Teachers' %}
+    ,
```

Comment:
Does not needs to be done in this PR: we should refactor the analytics configuration to be a Python dict in the template which is then printed with `config|tojson` - that's a little less verbose.